datasets/ONS-foreign-direct-investment-involving-uk-companies/main.py

```diff
@@ -265,24 +273,36 @@ def split_overrides(bag, splits):
     "gulf-arabian": "gulf-arabian-countries",
     "other-asian": "other-asian-countries"
 }
-observations["FDI Area"] = observations["FDI Area"].map(lambda x: fix.get(x, x))
+observations['FDI Area'] = observations['FDI Area'].map(lambda x: fix.get(x, x))
 
 
 # +
-
-inward_scraper.dataset.title = inward_scraper.dataset.title.replace(': inward', '')
-inward_scraper.dataset.comment = inward_scraper.dataset.comment.replace(
```

Comment:
add the comment created here to the catalog_metadata created below.

Reply:
Added in catalogue metadata as `summary = "Annual statistics on the investment of foreign companies into the UK, including for investment flows, positions and earnings into the UK and of UK companies abroad."`